hw/ip/spi_host/doc/theory_of_operation.md

```diff
@@ -431,6 +431,7 @@ The SPI_HOST supports interrupts for the following SPI events:
 - `RXFULL`: The SPI_HOST has run out of room in the RXFIFO.
 - `RXWM`: The number of 32-bit words in the RXFIFO currently exceeds the value set in [`CONTROL.RX_WATERMARK`](../data/spi_host.hjson#control).
 - `TXEMPTY`: The SPI_HOST has transmitted all the data in the TX FIFO.
+  Note the transmit FIFO may be empty while there is still one packet stored in `spi_host_byte_select`.
```

Comment:
The person writing the SW driver might not know what `spi_host_byte_select` refers to, since it is an internal submodule.

How about 
```
Note the transmit FIFO may be empty while there is still one packet pending in 
the internal transmit datapath (inside the `spi_host_byte_select` module). 
```
?

It probably makes sense to align the Hjson with this as well.

Reply:
Thanks, that is much better. I've taken your changes.